docs/sync.js

```diff
@@ -270,6 +270,54 @@ class User {
      */
     static register(server, username, password, callback) {}
 
+    /**
+     * Request a password reset email to be sent to a user's email.
+     * This will not throw an exception, even if the email doesn't belong to a Realm Object Server user.
+     * 
+     * This can only be used for users who authenticated with the 'password' provider, and passed a valid email address as a username.
+     * 
+     * @param {string} server - authentication server
+     * @param {string} email - The email that corresponds to the user's username.
+     * @return {Promise<void>} A promise which is resolved when the request has been sent.
+     */
+    static requestPasswordReset(server, email) {}
+    
+    /**
+     * Complete the password reset flow by using the reset token sent to the user's email as a one-time authorization token to change the password.
+     * 
+     * By default, the link that will be sent to the user's email will redirect to a webpage where, they can enter their new password.
+     * If you wish to provide a native UX, you may wish to modify the url to use deep linking to open the app, extract the token,
```

Comment:
How? There's no parameter here that allows customizing the url so I'm guessing it's configured somewhere else, and this should say where that somewhere else it.